v1/nix/pkgs/fetchPipMetadata/src/fetch_pip_metadata/lock_file_from_report.py

```diff
@@ -10,13 +10,17 @@
 )
 
 
-def git_repo_root():
-    return subprocess.run(
+def git_repo_root(directory):
```

Comment:
Supporting non-git users is good, but we should probably rename this to "repo_root" or something if it might not be git-specific?